### DISCUSSION TITLE
Add renovate.json - Onboarding MR and sementic commits

### DISCUSSION THREAD

**Discussion by MollardMichael** [renovatebot/renovate#8667] [Request Help]
Hello,

I am enforcing semantic commits on my repo (no commits can be pushed or merged without fitting with the sementic commit guidelines).

The issue is that the onboarding MR is not following semantic versioning at all.
Can I do something about it ? Is there an environment variable I can give renovate (I'm using the cli) to behave as I want hi to behave ?

**Comment by viceice** [comment]
sure, you should be able to verride the onboarding config to force `semanticcommits=true` in bot config and onboarind config

https://docs.renovatebot.com/configuration-options/#semanticcommits

**Reply by rarkins** [reply]
Self hosted or app?

**Reply by MollardMichael** [reply]
Self Hosted. I had to amend the commit and change the commit message myself.

Renovate is looking at commit history and do provide sementic commits for all the following dependencies update but it's not doing so for the onboarding MR.

**Reply by rarkins** [reply]
What's your bot config? Ie the config.js
